allyapp/car_api
# Code challenge

PR for Code Challenge Submission.

Originally done in separate repo, but forked to this repo (my mistake). I added instructions to the README; basic steps whenever you clone down the repo. 

- Just run "bundle"
- Run "rake db:setup"
- To test, run "rake"
- To see it in the browser, run "rails s"

At first, I tried GeoKit, but had trouble with implementing it in Rails. It seems the docs and Stack Overflow have some conflicting directions to follow, and the methods I wanted to use were deprecated. Sad days. 

So I switched to Geocoder and it seems to be working, unless I input bogus lat-long coordinates. Based on the seed file, if I input lat-long coordinates too far off, I will only return an empty array. This is what I wanted to happen, but took some time to understand this fact. It was mostly because of my ignorance of how lat-long works regards to distance.
